dev/vagrant-config/scripts/php.sh

```diff
@@ -15,7 +15,7 @@ sed -i 's/group = www-data/group = vagrant/g' /etc/php/7.0/fpm/pool.d/www.conf
 echo 'error_log = /vagrant/storage/logs/php_errors.log' >> /etc/php/7.0/fpm/php.ini
 echo 'xdebug.remote_enable = on' >> /etc/php/7.0/fpm/conf.d/20-xdebug.ini
 echo 'xdebug.remote_connect_back = on' >> /etc/php/7.0/fpm/conf.d/20-xdebug.ini
-echo 'xdebug.remote_host = 192.168.35.22' >> /etc/php/7.0/fpm/conf.d/20-xdebug.ini
+echo 'xdebug.remote_host = 192.168.25.1' >> /etc/php/7.0/fpm/conf.d/20-xdebug.ini
```

Comment:
`xdebug.remote_connect_back` will automatically connect to the IP of the originating request, specifying remote_host isn't necessary as per the docs it's ignored.

https://xdebug.org/docs/all_settings#remote_host